requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
```

Comment:
I forgot this part in `setup.py`:

```python
with open('requirements.txt') as f:
    require = f.readlines()
install_requires = [r.strip() for r in require]
...
install_requires=install_requires,
...

```

I like the `requirements.txt`, but I have no strong feelings about it.  Adding the requirements to the `setup.py` file is also fine.  Let me know which way you want to go and I will fix the PR.